src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        return "Certain machine started its work";
```

Comment:
The method 'doWork' should not return a string. Instead, it should print an informative message to the console. Also, specify what type of machine is working, e.g., 'Excavator started its work'.